packages/openapi3-parser/lib/parser.js

```diff
@@ -104,7 +104,7 @@ const deduplicateUnsupportedAnnotations = R.curry((namespace, parseResult) => {
     },
     R.T);
 
-  const result = new namespace.elements.ParseResult(R.filter(filterWarnings, parseResult));
+  const result = R.filter(filterWarnings, parseResult);
```

Comment:
As fantasy land filter would result in the same type, we no longer need to wrap it back into a parse result.